pkg/minishift/registration/redhat_test.go

```diff
@@ -112,22 +104,14 @@ func TestRedHatRegistratorUnregister(t *testing.T) {
 	}
 	commander := provision.GenericSSHCommander{Driver: d}
 	registrator := NewRedHatRegistrator(commander)
-	s.CommandToOutput["sudo subscription-manager version"] = `server type: This system is currently not registered.`
-	if err := registrator.Unregister(param); err != nil {
-		t.Fatal("Distribution shouldn't able to unregister")
-	} else {
-		cmd := "sudo subscription-manager unregister"
-		if _, ok := s.Commands[cmd]; ok{
-			t.Fatalf("Expected command: %s", cmd)
-		}
-	}
+
 	s.CommandToOutput["sudo subscription-manager version"] = `server type: RedHat Subscription Management`
 	if err := registrator.Unregister(param); err != nil {
 		t.Fatal("Distribution should able to unregister")
 	} else {
 		cmd := "sudo subscription-manager unregister"
 		if _, ok := s.Commands[cmd]; !ok{
-			t.Fatalf("Expected command: %s", cmd)
+			t.Fatalf("Expected command: sudo subscription-manager unregister")
```

Comment:
??? 

the content of _cmd_ is your expected command! This is the key you go into the commands map. If you want to show what actually got received you need to change the test. The expected command as per your test is whatever you put into 'cmd'.